tests/h/util/metrics_test.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals
+import pytest
+from mock import call
+from h.util import metrics
+
+from webob.multidict import MultiDict
```

Comment:
This third-party import should go before the local imports above (see https://www.python.org/dev/peps/pep-0008/#imports)